metricflow/model/semantics/metric_semantics.py

```diff
@@ -145,30 +144,38 @@ def metric_input_specs_for_metric(
             original_metric_obj = self.get_metric(input_metric.as_reference)
 
             # This is the constraint parameter added to the input metric in the derived metric definition
-            combined_spec_constraint = (
-                WhereConstraintConverter.convert_to_spec_where_constraint(
-                    data_source_semantics=self._data_source_semantics,
-                    where_constraint=input_metric.constraint,
+            combined_filter = (
+                ResolvedWhereFilter.create_from_where_filter(
+                    where_filter=input_metric.constraint,
+                    column_association_resolver=column_association_resolver,
                 )
                 if input_metric.constraint is not None
                 else None
             )
 
             # This is the constraint parameter included in the original input metric definition
             if original_metric_obj.constraint:
-                original_metric_constraint = WhereConstraintConverter.convert_to_spec_where_constraint(
-                    data_source_semantics=self._data_source_semantics,
-                    where_constraint=original_metric_obj.constraint,
+                # original_metric_constraint = WhereConstraintConverter.convert_to_spec_where_constraint(
+                #     data_source_semantics=self._data_source_semantics,
+                #     where_constraint=original_metric_obj.constraint,
+                # )
+                # combined_spec_constraint = (
+                #     combined_spec_constraint.combine(original_metric_constraint)
+                #     if combined_spec_constraint
+                #     else original_metric_constraint
+                # )
```

Comment:
We should probably remove these if you haven't already.

Reply:
Oh yeah, will remove.